alpha/apps/kaltura/lib/batch2/bcdl/kBusinessConvertDL.php

```diff
@@ -300,15 +300,15 @@ public static function setAsDefaultThumbAsset($thumbAsset)
 			$thumbAsset->addTags(array(thumbParams::TAG_DEFAULT_THUMB));
 			$thumbAsset->save();
 			KalturaLog::info("Setting entry [". $thumbAsset->getEntryId() ."] default ThumbAsset to [". $thumbAsset->getId() ."]");
-		}
 
-		$entry->setThumbnail(".jpg");
```

Comment:
If the thumbnail is already a `default` one, the code starting from this line was being executed and adding more similar entries into file_sync table. So moved it up under the IF statement